ext/standard/info.c

```diff
@@ -637,12 +640,14 @@ PHPAPI void php_print_info(int flag)
 	        the_time = time(NULL);
 	        ta = php_localtime_r(&the_time, &tmbuf);
 
-            php_info_print("<a href=\"http://www.php.net/\"><img border=\"0\" src=\"");
-	        if (ta && (ta->tm_mon==3) && (ta->tm_mday==1)) {
-		        php_info_print(PHP_EGG_LOGO_DATA_URI "\" alt=\"PHP logo\" /></a>");
-	        } else {
-		        php_info_print(PHP_LOGO_DATA_URI "\" alt=\"PHP logo\" /></a>");
-			}
+			php_info_print("<a href=\"http://www.php.net/\"><img border=\"0\" src=\"");
```

Comment:
Speaking of using stylesheets. `highlight_*()` from what I can see in the commit it is now using the style attr. If you are targetting 7.1 then you could use classes instead and use http://caniuse.com/#feat=style-scoped *(provided there is enough browser support by then)*.